content/r/concepts/variables/variables.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'Variables'
+Description: 'A variable is used to store data that can be accessed later by subsequent code.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Variables'
+  - 'Integers'
+  - 'Booleans'
+  - 'Strings'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+A variable is used to store data that can be accessed later by subsequent code. In `R` there are no variable "declaration" commands.
+Variables are instead created with the assignment operator, `<-`. (The more familiar assignment operator, `=`, can be used instead, but is discouraged.)
+
+```r
+# this creates a variable containing the string "value"
+variable <- "value"
+```
+
+The assignment operator can be chained together to initialize multiple variables at once:
```

Comment:
By "chained" do we mean that all the `varx` variables are assigned the same value of 0? Or does it mean that `var3` is directly set to 0, `var2` is set to `var3`, and `var1` is set to `var2`?

Reply:
@Dusch4593: I'm not sure of the exact mechanics of this syntax, but it would make sense that the interpreter does the latter. Since the result is effectively the same, do we need to go into the distinction?